core/io/resource_format_binary.cpp

```diff
@@ -287,7 +287,7 @@ Error ResourceInteractiveLoaderBinary::parse_variant(Variant &r_v) {
 				case OBJECT_INTERNAL_RESOURCE: {
 					uint32_t index = f->get_32();
 					String path = res_path + "::" + itos(index);
-					RES res = ResourceLoader::load(path);
+					RES res = ResourceLoader::load(path, "", no_subresource_cache);
```

Comment:
IMO internal resources should be cached in some internal Map<> and loaded from there. This is actually always using the resource cache, so it will not work.